package.json

```diff
@@ -47,7 +53,8 @@
   },
   "scripts": {
     "start": "./node_modules/.bin/grunt start",
-    "devSetup": "./node_modules/.bin/bower install && ./node_modules/.bin/grunt install"
+    "devSetup": "./node_modules/.bin/bower install && ./node_modules/.bin/grunt install",
```

Comment:
Does it depend on npm version? I remember that it was not working with earlier version of npm ~ 1.x if grunt/bower are not installed globally.